lib/stats.js

```diff
@@ -67,7 +67,7 @@ var stats = {
       var childrens = options.childrens ? stat.cutime + stat.cstime : 0
       var total = 0;
 
-      if(history.utime) {
+      if(typeof history.utime !== 'undefined') {
```

Comment:
Hmm I don't understand, if `history.utime = undefined`, it'll still follow the else path, won't it? Also, if it's `0` those should be the same `(stat.stime - history.stime) + (stat.utime - history.utime) + childrens` == `stat.stime + stat.utime + childrens`. Btw, `history.utime` should never be `undefined`.